config/locales/account/en.yml

```diff
@@ -3,7 +3,7 @@ en:
   account:
     connected_apps:
       associated: Connected %{timestamp}
-      description: With your login.gov account, you can securely connect to multiple
+      description: With your Login.gov account, you can securely connect to multiple
```

Comment:
note for a followup ticket, we should probably update this to use the APP_NAME constant like `%{app_name}` like we do in other strings

Reply:
I like this idea.

@zachmargolis what do you think the frequency of interchangeability is? I like the idea of it being a var, but am not sure how often it will change in the future.

Reply:
For me it's more about consistency, that half our strings have `%{app_name}` and half don't, I also hope that we wont change the formatting of "Login.gov" much in the future

Reply:
Maybe a good topic for the Engineering Huddle?

Reply:
Filed https://cm-jira.usa.gov/browse/LG-4886